packages/graphiql/src/css/app.css

```diff
@@ -161,14 +161,14 @@
   position: relative;
 }
 
-.graphiql-container .variable-editor {
+.graphiql-container .extra-editor {
```

Comment:
let's use the same class names as before to keep parity, and add new ones as needed

Reply:
@acao as in have both `variable-editor` and `extra-editor` here, or just use `variable-editor`?

Reply:
we should make sure the actual dom elements have *both* classes, and you can use the new classes here. in the new 2.0.0 RFC, we don't use CSS classes

Reply:
👍 updated